api/handlers/app_test.go

```diff
@@ -525,6 +530,133 @@ var _ = Describe("App", func() {
 		})
 	})
 
+	Describe("GET /v3/apps/{guid}/manifest", func() {
+		var dropletRecord repositories.DropletRecord
+		BeforeEach(func() {
+			processRecord := repositories.ProcessRecord{
```

Comment:
This `BeforeEach` is doing quite a lot of initialisation on records. Do all those values make it to the manifest eventually? If not, we could get rid of the unused to minimise the unused boilerplate